app/[id]/share.tsx

```
@@ -0,0 +1,85 @@
+    const handleCopy = async () => {
+        const shareLink = link;
+        await Clipboard.setStringAsync(shareLink);
+        Toast.info("Lien copié dans le presse-papier");
+    };
+
+
+    return (
+        <View style={styles.container} className="p-4">
+            {/* // Header */}
+            <Text className="text-2xl font-bold dark:text-white text-center mb-2">
+                Partager "{trip?.name}"
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&`#34`;`, `&rdquo;`.

(react/no-unescaped-entities)